### PR TITLE
some kindly-meant unsolicited front-end fixes

### PR DESCRIPTION
**Pull request by sknep**
A gift for you, should you want it. More can be given :)

### CODE REVIEW

File: src/_includes/components/macros.njk [hunk 1]
```diff
@@ -14,14 +14,8 @@
     </p>
   </div>
 </div>
-{% endmacro %}
+{%- endmacro -%}
 
-{% macro link(key, text) %}
-  {% set url = "" %}
-    {% for k, v in glossary %}
```
Comment (sknep) [inline src/_includes/components/macros.njk:21]: You don't have access to the data variables in a nunjucks macro, especially ones added through yaml. If you want something like this, we'd have better luck if it was JS/JSON.

File: src/pages/login.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+---
+layout: page.njk
+title: Logging into the FAC
+tags:
+  - grantees
+  - auditors
+---
+
```
Comment (sknep) [inline src/pages/login.md:8]: Might I recommend markdown for content?

File: src/_includes/components/macros.njk [hunk 3]
```diff
@@ -1,4 +1,4 @@
-{% macro applies_to(applies, type='info') %} {# body, title='', type='info') #}
```
Comment (sknep) [inline src/_includes/components/macros.njk:1]: These dashes remove extra whitespace, which is less important for these block-level template functions but essential for inline ones like glossary links, etc

File: src/_includes/layout.njk [hunk 4]
```diff
@@ -11,11 +11,9 @@
 </head>
 
 <body>
-  <main id="main-content">
   {% include 'header.njk' %} 
-  <div class="grid-container"  style="margin: 10px;">
```
Comment (sknep) [inline src/_includes/layout.njk:16]: [ಠ_ಠ]

**Comment by jadudm** [comment]
Awesome. Thank you!

**Pull request merged**